client/src/app/modules/model-manager/components/omz-import-ribbon-content/omz-import-ribbon-content.component.html

```diff
@@ -65,29 +73,62 @@
 
 <ng-template #filterGroupTemplate>
   <wb-model-zoo-filter-group [formControl]="filtersControl" *ngIf="dataSource.filtersOptions$ | async as filterOptions">
-    <wb-model-zoo-filter group="task_type">
-      <wb-model-zoo-filter-title>Task Type</wb-model-zoo-filter-title>
-      <wb-model-zoo-filter-option *ngFor="let option of filterOptions.taskType" [value]="option" icon="list_alt">
-        {{ option }}
-      </wb-model-zoo-filter-option>
-    </wb-model-zoo-filter>
+    <ng-container *wbSkeleton="isLoading; repeat: 7; width: 'rand'; height: '40px'; gap: '20px'">
```

Comment:
The same question as in https://github.com/openvinotoolkit/workbench/pull/50/files#r836909111 . Let's discuss